suse_openstack_cloud

```diff
@@ -395,14 +395,14 @@ fi
 ###############################################################
 section_header "RabbitMQ broker"
 
-if [ -e /usr/sbin/rabbitmqctl ]; then
-    rabbitmq_tmpdir=$(mktemp -d)
-    rabbitmq_report_log=$rabbitmq_tmpdir/rabbitmq_report.log
+rabbitmqctl=/usr/lib64/rabbitmq/bin/rabbitmqctl
```

Comment:
@dirkmueller : Especially for you, I've debugged once again issue from https://github.com/SUSE-Cloud/supportutils-plugin-suse-openstack-cloud/pull/33
So, error 
```
Error: could not recognise command
```
is caused because supportconfig is trying to call `/usr/lib64/rabbitmq/bin/rabbitmqctl` with wrong arguments, like follows
```
/usr/lib64/rabbitmq/bin/rabbitmqctl /usr/lib/supportconfig/plugins/suse_openstack_cloud "report"
```